rflx/templates/rflx_generic_types.adb

```diff
@@ -40,6 +40,29 @@ is
 
    use {prefix}RFLX_Arithmetic;
 
+   procedure Get_Index_Offset (First, Last : Long_Integer; Off : Offset; Value_Size : Positive;
+                               RME_Index : out Index; LME_Index : out Index; RME_Size : out Natural; LME_Size : out Natural)
```

Comment:
Could you please make the formatting of the subprogram signature consistent to the other subprograms in this file?

Reply:
done